backends-velox/src/main/scala/org/apache/gluten/execution/ColumnarPartialProjectExec.scala

```diff
@@ -332,13 +336,27 @@ object ColumnarPartialProjectExec {
     }
   }
 
-  def create(original: ProjectExec): ProjectExecTransformer = {
-    val replacedAliasUdf: ListBuffer[Alias] = ListBuffer()
-    val newProjectList = original.projectList.map {
-      p => replaceExpressionUDF(p, replacedAliasUdf).asInstanceOf[NamedExpression]
+  def create(original: SparkPlan): UnaryTransformSupport = {
+    val transformedPlan = original match {
+      case p: ProjectExec =>
+        val replacedAliasUdf: ListBuffer[Alias] = ListBuffer()
+        val newProjectList = p.projectList.map {
+          p => replaceExpressionUDF(p, replacedAliasUdf).asInstanceOf[NamedExpression]
+        }
+        val partialProject =
+          ColumnarPartialProjectExec(p, p.child)(replacedAliasUdf.toSeq)
+        ProjectExecTransformer(newProjectList, partialProject)
+      case f: FilterExec =>
+        val replacedAliasUdf: ListBuffer[Alias] = ListBuffer()
+        val newCondition = splitConjunctivePredicates(f.condition)
```

Comment:
Does the replaceExpressionUDF can handle AND? What if the filter condition contains `or`?

Reply:
It can works under `and`, I add one more ut for this.
For `OR`, I think you are right, it's not safe to pre evalute where there are `OR` because of short-circuit evalution. I'll fix it

Reply:
I'm not fully understand here, why we need to splitConjunctivePredicates and then combine with AND, what if we remove it?